atlas/getting-started/deploy-from-existing-repo.mdx

```diff
@@ -25,7 +23,12 @@ For example, the `package.json` file for a Faust.js or Next.js app would need to
   },
 ```
 
-If you don't know what command needs to be run to build your app, please refer to your JavaScript framework's documentation.
+3. creating a "ATLAS_BUILD_SCRIPT" environmental variable in the user-portal and adding the command as value. Learn more about environmental variables [here](#env-var).
```

Comment:
@dgallardox I think we should capitalize the first letter of this sentence and drop the "ing", put `ATLAS_BUILD_SCRIPT` in backticks since it's inline code, change "user-portal" to "User Portal" since it's a proper noun, and simplify the wording a bit. So this will become:

3. Create an `ATLAS_BUILD_SCRIPT` environmental variable in the User Portal, with the command to be run as its value. Learn more about environmental variables [here](#env-var).